### DISCUSSION TITLE
Creation of node in BMC

### DISCUSSION THREAD

**Discussion by TannaManohar** [ipmitool/ipmitool#325] [Q&A]
Hi
I am new to IPMI, and my BMC is MSP430F5358
Q1) How to set up my ipmi_ssif in the host(Linux) so that it can communicate with my external BMC server via SMBus?
Q2 What will be my requirements to set up my BMC for the ipmitool communication?

**Comment by AlexanderAmelkin** [comment]
The prerequisite on Linux is generally existence of `/dev/ipmi0` (for `ipmitool -I open`, which is the default).
How to set up the drivers in Linux is unfortunately out of the scope of this project. I believe that can be googled up.
